CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
  - #3171, #3046, Log schema cache stats to stderr - @steve-chavez
  - #3210, Dump schema cache through admin API - @taimoorzaeem
  - #2676, Performance improvement on bulk json inserts, around 10% increase on requests per second by removing `json_typeof` from write queries - @steve-chavez
+ - #3248, Add CLI page to docs - @taimoorzaeem
+   + Add config type to ``--example`` in CLI
```

Comment:
This should mention that `--example` now accepts the `file,db,env` parameters.

Also, the "Add CLI page to docs" should be in a separate line, I think. We used to have a [Documentation Improvements](https://postgrest.org/en/v10/releases/v10.0.0.html#documentation-improvements) section in the docs. Maybe a new `### Docs` subtitle is needed here?